pkg/client/restclient/client_test.go

```diff
@@ -110,8 +110,8 @@ func TestDoRequestCreated(t *testing.T) {
 	if err != nil {
 		t.Fatalf("unexpected error: %v", err)
 	}
-	created := false
-	body, err := c.Get().Prefix("test").Do().WasCreated(&created).Raw()
+	var statusCode int
+	body, err := c.Get().Prefix("test").Do().StatusCode(&statusCode).Raw()
```

Comment:
Ok. Now I'm really confused. `created` and now `statusCode` don't really seem to be used/validated ANYWHERE. In `validate`, there's a check against `testParam.actualCreated`, and I can't find anywhere where that variable is set.

Shouldn't these `TestParam`s have a field initializer `actualCreate: statusCode == http.StatusCreated,` in them?